CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## v0.7.0 / 2017-05-19
```

Comment:
So apparently  there were discussions in the past about when to use the `v` prefix and when not and the decision was to only use it in ambiguous contexts like git tags, but not in changelogs or releases. For consistency with all other prometheus projects, let's remove it here.